file-formats/sito/type/zif_aff_sito_v1.intf.abap

```diff
@@ -0,0 +1,551 @@
+    "! SIT2_DO_VH_SRV_PROTOCOL_VER
+    BEGIN OF co_sit2_do_vh_srv_protocol_ver,
+      "! <p class="shorttext">V2</p>
+      "! V2
+      v2 TYPE ty_sit2_do_vh_srv_protocol_ver VALUE '01',
+      "! <p class="shorttext">V4</p>
+      "! V4
+      v4 TYPE ty_sit2_do_vh_srv_protocol_ver VALUE '02',
+    END OF co_sit2_do_vh_srv_protocol_ver.
+
+  "! <p class="shorttext">SIT2_DO_VH_SRV_PATH_TYPE</p>
+  "! SIT2_DO_VH_SRV_PATH_TYPE
+  "! $values {@link zif_aff_sito_v1.data:co_sit2_do_vh_srv_path_type}
+  TYPES ty_sit2_do_vh_srv_path_type TYPE n LENGTH 2.
+
+  CONSTANTS:
+    "! <p class="shorttext">SIT2_DO_VH_SRV_PATH_TYPE</p>
+    "! SIT2_DO_VH_SRV_PATH_TYPE
+    BEGIN OF co_sit2_do_vh_srv_path_type,
+      "! <p class="shorttext">Standard</p>
+      "! Standard
+      standard TYPE ty_sit2_do_vh_srv_path_type VALUE '01',
+      "! <p class="shorttext">Custom</p>
+      "! Custom
+      custom   TYPE ty_sit2_do_vh_srv_path_type VALUE '02',
+    END OF co_sit2_do_vh_srv_path_type.
+
+  "! <p class="shorttext">SIT2_DO_VH_APP_TYPE</p>
+  "! SIT2_DO_VH_APP_TYPE
+  "! $values {@link zif_aff_sito_v1.data:co_sit2_do_vh_app_type}
+  TYPES ty_sit2_do_vh_app_type TYPE n LENGTH 2.
+
+  CONSTANTS:
+    "! <p class="shorttext">SIT2_DO_VH_APP_TYPE</p>
+    "! SIT2_DO_VH_APP_TYPE
+    BEGIN OF co_sit2_do_vh_app_type,
+      "! <p class="shorttext">All Apps</p>
+      "! All Apps
+      all_apps TYPE ty_sit2_do_vh_app_type VALUE '01',
+    END OF co_sit2_do_vh_app_type.
+
+  "! <p class="shorttext">SIT2_DO_VH_TYPE</p>
+  "! SIT2_DO_VH_TYPE
+  "! $values {@link zif_aff_sito_v1.data:co_sit2_do_vh_type}
+  TYPES ty_sit2_do_vh_type TYPE n LENGTH 2.
+
+  CONSTANTS:
+    "! <p class="shorttext">SIT2_DO_VH_TYPE</p>
+    "! SIT2_DO_VH_TYPE
+    BEGIN OF co_sit2_do_vh_type,
+      "! <p class="shorttext">Service</p>
+      "! Service
+      service TYPE ty_sit2_do_vh_type VALUE '01',
+    END OF co_sit2_do_vh_type.
+
+  TYPES:
+
+    "! <p class="shorttext">Texts</p>
+    "! Situation object structue
+    BEGIN OF ty_sit2_obj_struc_t,
+      "! <p class="shorttext">Description</p>
+      "! Situation object structure description
+      description TYPE c LENGTH 255,
+    END OF ty_sit2_obj_struc_t,
+
+    "! <p class="shorttext">Texts</p>
+    "! Situation object structue
+    ty_sit2_obj_struc_t_list TYPE STANDARD TABLE OF ty_sit2_obj_struc_t WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Semantic Keys</p>
+    "! Semantic keys of an object structure
+    BEGIN OF ty_sit2_obj_str_sk,
+      "! <p class="shorttext">Field Order</p>
+      "! Object field order for situation
+      field_order TYPE n LENGTH 2,
+      "! <p class="shorttext">Field</p>
+      "! Object field
+      field       TYPE c LENGTH 30,
+    END OF ty_sit2_obj_str_sk,
+
+    "! <p class="shorttext">Semantic Keys</p>
+    "! Semantic keys of an object structure
+    ty_sit2_obj_str_sk_list TYPE STANDARD TABLE OF ty_sit2_obj_str_sk WITH DEFAULT KEY,
```

Comment:
If you want to keep the field `field_order`, should the entries be sorted by the field_order?

Reply:
for the moment, we are fine with keeping field field_order, based on standard table
(we tried to use sorted table, but we obtain a syntax error in if_aff_sito_v1)

Reply:
If you want to have a sorted table, it should be possible to have it like 
```suggestion
     ty_sit2_obj_str_sk_list TYPE SORTED TABLE OF ty_sit2_obj_str_sk WITH UNIQUE KEY field_order,
```